src/components/field-key/list.vue

```diff
@@ -227,3 +223,29 @@ export default {
   }
 }
 </style>
+
+<i18n lang="json5">
+{
+  "en": {
+    "action": {
+      "create": "Create App User"
+    },
+    "heading": {
+      "full": "App Users in this Project only will be able to download and use Forms within this Project. When you create a new App User, it will not have access to any Forms at first. To set the Forms each App User may access, use the {formAccess} tab. Multiple devices can use the same App User profile without problem. For more information, {clickHere}.",
+      "formAccess": "Form Access",
```

Comment:
It seems this should always be the same as the tab name. Is it impractical to use the same resource?

Reply:
I was worried about a change due to the accusative case, but maybe "tab" would receive the accusative marker, but "Form Access" wouldn't? Also, since "the Form Access tab" is a definite noun phrase, I think construct state might be used in Arabic or Hebrew that would change the spelling of "Form Access": https://en.wikipedia.org/wiki/I%E1%B8%8D%C4%81fah. But I don't know enough to know how likely that is here.